<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <widget class="QMenu" name="menuAbrir_Ventana">
     <property name="title">
      <string>Abrir Ventana</string>
     </property>
     <addaction name="actionAbrir_Widget_2"/>
     <addaction name="actionAbrir_Dialog_2"/>
    </widget>
    <addaction name="menuAbrir_Ventana"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir_Widget">
   <property name="text">
    <string>Abrir Widget</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAcerca_de_2">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionAcerca">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionAbrir_Widget_2">
   <property name="text">
    <string>Abrir Widget</string>
   </property>
  </action>
  <action name="actionAbrir_Dialog_2">
   <property name="text">
    <string>Abrir Dialog</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
